nb/api/Line.md

```diff
@@ -41,11 +41,13 @@ Line(5)
 Line(5).and(rz(1 / 8)) produces a 1/8th angle.
 
 ```JavaScript
-Line(seq({ from: -1, upto: 1, by: 1 / 8 }, noOp, List))
+Line(seq({ from: -1, upto: 1, by: 1 / 8 }, noOp(), List))
```

Comment:
Why the change to have noOp called instead of passed as a function? I think that the old way makes more sense to me intuitively

Reply:
noOp is a => b => b. So noOp() gets us the b => b which does what we want to pass through the numbers to gather with List.

But maybe this is a bit confusing -- might need to rethink it.

Reply:
Ah I understand now. It's a little confusing, but I don't think it's a big deal